Starbust.md

```diff
@@ -0,0 +1,162 @@
+# Fraud detection in RHODS using Starbust Enterprise
+
+##### Requirements:
+
+- RHODS working cluster with administrator access to it
+- Startbust Enterprise licence
+- Write access to an Amazon S3 bucket
+- Access to the original dataset
+
+#### What is Trino and Starbust Enterprise?
+
+Trino (formerly Presto® SQL) is the fastest open source, massively parallel
+processing SQL query engine designed for analytics of large datasets distributed
+over one or more data sources in object storage, databases and other systems.
+
+Starburst Enterprise platform (SEP) is a fully supported, enterprise-grade
+distribution of Trino. It adds integrations, improves performance, provides
+security, and makes it easy to deploy, configure and manage your clusters.
+
+### Goal
+
+In this document we will cover how to work with the data of the fraud-detection
+example with SQL-like queries and an instance of SEP running in the Red Hat
+Openshift Data Science cluster.
+
+#### Store the original data in S3
+
+Put the original file with raw data in some persistent storage like 
+Amazon Simple Storage Service (S3).
+
+Upload the
+file [creditcard_with_empty_values.csv]('https://drive.google.com/file/d/1YhmV3vPbFe-JXU_biwvaizV0WGhAegH1/view?usp=share_link')
+to a bucket called `rhods-fraud-detection/data`. The RHODS sandbox cluster **must** have connectivity and
+write access to the bucket.
+
+#### Working from the SEP WebUI
+
```

Comment:
we need to provide yamls to this repo and then oc commands to deploy the startburst and hive instances.
yamls: 
1. Secret SEP
2. Secret AWS
3. SEP Configuration
4. Hive Configuration
5. Route exposing SEP

commands:
1. oc apply secret SEP
2. oc apply secret AWS
3. oc patch csv of starburst with the secret SEP.
4. oc apply SEP config
5. oc aply hive config
6. oc apply route